README.md

```diff
@@ -34,14 +35,28 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic HTML gives a meaning to the web pages parts instead of just presenting them.
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+display: inline; allows to set a width and height 
+display: block; top and bottom margin and padding is respected
+
 3. What are the 4 areas of the box model?
 
+content
+padding
+border
+margin
+
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
 
+This property works on the content on the cross-axis
+
 5. Explain why git is valuable to a team of developers.
 
+With Git you can build a protocoll of all changes made on files so you can track every version of the code you are working on. You can see who made the last changes on a file and can bring the file to the old state. 
```

Comment:
Nice!! it is also super useful while collaborating with other programmers on the same project